finan/models/finan_retorno.py

```diff
@@ -705,6 +705,25 @@ def processar_retorno(self):
         # Processa os boletos do arquivo
         for boleto in arquivo_retorno.boletos:
 
+            # verifica se o boleto é duplicado
+            if boleto.pagamento_duplicado == True:
```

Comment:
Você pode colocar "if boleto.pagamento_duplicado:" direto, não precisa da comparação com True. Igualmente no elif, pode-se usar "if not boleto.pagamento_duplicado:" assim o código fica mais simples.